src/nmodl/init.cpp

```diff
@@ -153,6 +154,11 @@ static const char* extdef5[] = {/* the extdef names that are not threadsafe */
 #include "extdef5.h"
                                 0};
 
+/* random to nrnran123 functions */
+std::map<std::string, const char*> extdef_rand = {
+#include "extdef_rand.h"
```

Comment:
Instead of using this `include` pattern, I would remove `extdef_rand.h` and copy those variables here. It's not used elsewhere.